data-collection/features/index.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 4
+title: Features
+---
+
+** The Huma Experience **
+
+Huma is transforming healthcare and research globally. Our digital ‘hospital at home’ (RPM), decentralized clinical trial platform (DCT), and software as a medical device (SaMD) use real-time health data from smartphones to help patients, clinicians, researchers, and healthcare systems. We help people live longer, fuller lives.
+
+The features Huma provides takes care of giving the user control and ownership of all data and information provided. Included is the ability to accept and reject evolving consent and be confident in a secure, yet intuitive product. 
```

Comment:
The features Huma provides helps Care Teams and Research Team leverage on delivering care and conducting research remotely.  They also give patients control and ownership of all data and information provided. Included are the ability to accept and reject evolving consent, data flagging capability to highlight critical data and be able to send one-way messages to patients/participants.